tutorials/Consistent_Scores.ipynb

```diff
@@ -0,0 +1,465 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Consistent scores\n",
+    "\n",
+    "A \"consistent scoring function\" is a scoring function where a forecaster's expected score will be optimised by following a forecast directive. An example of a forecast directive is to \"predict the mean (i.e., expected) rainfall amount\". Using a consistent scoring function ensures that a forecaster or forecast system developer isn't faced with the dilemma of whether to \n",
```

Comment:
... by following a forecast directive, or, where a model's score will be optimised by minimising its objective (error) function.

Reply:
An example of the equivalent objective function is for a machine learning model to minimise the mean absolute error on the validation set.

Reply:
I think that's reasonable. I think maybe more could be said but I can't think of anything better, so let's just leave it.